gatsby-config.js

```diff
@@ -20,6 +26,15 @@ module.exports = {
     'gatsby-plugin-eslint',
     'gatsby-plugin-remove-trailing-slashes',
     'gatsby-plugin-netlify',
+    {
+      resolve: `gatsby-plugin-algolia`,
+      options: {
```

Comment:
Can we move the plugin definition to outside the config and make the entire plugin conditional on?

 `typeof process.env. ALGOLIA_ADMIN_KEY !== 'undefined' && (process.env.BRANCH === 'master' || process.env.CIRCLECI)`?

This means the algolia index will only get updated if the site is being built on our CI or in Netlify on master branch. The reason for this is we don't want to have to pass out the admin key to all developers, and after we go live any dev doing work on this is going to get their own key only if they are doing search work.

Reply:
Of course. Do you know if it's best to retain the plugin's position or it doesn't matter?

Reply:
@kevee fixed